admin_guide/overcommit.adoc

```diff
@@ -45,68 +45,36 @@ lowest quality of service.
 [[tune-buffer-chunk-limit]]
 === Tune Buffer Chunk Limit
 
-If Fluentd logger is unable to keep up with a high number of logs, you will need
-to increase the compute resource values.
+If Fluentd logger is unable to keep up with a high number of logs, it will need
+to switch to file buffering to reduce memory usage and prevent data loss.
 
-The memory limit is used to calculate the Fluentd `buffer_queue_limit` as follows:
+The Fluentd `buffer_chunk_limit` is determined by the environment variable
+`BUFFER_SIZE_LIMIT`, which has the default value 8m. The file buffer size per
+output is determined by the environment variable `FILE_BUFFER_LIMIT`, which has
+the default value 256Mi. The permanent volume size has to be larger than
+`FILE_BUFFER_LIMIT` times number of output.
```

Comment:
By times number of output, do you mean multiplied by the `FILE_BUFFER_LIMIT` value?